### PR TITLE
fix: GKE creds issue

### PR DESCRIPTION
**Pull request by kumarnarendra701**
**- What I did**
 - Fix the GKE creds plugin issue
 
**- How I did it**
- Modified script to support new plugin

**- How to verify it**
- Verify by action last run

### CODE REVIEW

File: packages/at_root_server/README.md [hunk 1]
```diff
@@ -2,4 +2,6 @@
 
 # at_root_server
 
+
```
Comment (cpswan) [inline packages/at_root_server/README.md]: Why is this file even here, and how did it get extra empty lines?

File: .github/workflows/at_server_prod_deploy.yaml [hunk 2]
```diff
@@ -44,22 +44,26 @@ jobs:
...
-          service_account_key: ${{ secrets.GKE_SA_PROD_KEY }}
-          project_id: ${{ secrets.GKE_PROD_PROJECT }}
+          credentials_json: '${{ secrets.GKE_SA_PROD_KEY }}'
+
+      - name: 'Set up Cloud SDK'
+        uses: 'google-github-actions/setup-gcloud@d51b5346f85640ec2aa2fa057354d2b82c2fcbce # v1.0.1'  
 
       # Get the GKE credentials so we can deploy to the cluster
-      - uses: google-github-actions/get-gke-credentials@894c221960ab1bc16a69902f29f090638cca753f # v1.0.1
+      - uses: google-github-actions/get-gke-credentials@0a5f57b989b81d2725178cda257a00acf7d6bb5f # v0.7.0
```
Comment (cpswan) [inline .github/workflows/at_server_prod_deploy.yaml]: Why these different versions?

v1.0.1 initially (which is presumably latest), but then v1.0.0 for dev and v0.7.0 for prod

File: packages/at_root_server/README.md (hunk 1 above)
Comment (kumarnarendra701) [reply packages/at_root_server/README.md]: I added one space to trigger the action file.

Comment (cpswan) [reply packages/at_root_server/README.md]: For something that's not self explanatory from the file itself (like a white space change), please use more meaningful commit messages to say that's what you're doing (and mention it in the PR 'what I did' section).

File: .github/workflows/at_server_prod_deploy.yaml (hunk 2 above)
Comment (kumarnarendra701) [reply .github/workflows/at_server_prod_deploy.yaml]: Fixed version issue

**Review by cpswan — APPROVED** [review]

**Pull request merged**